event hubs migration guide.md

```diff
@@ -0,0 +1,81 @@
+# Migration Guide (EventHubs 2 to 5)
+
+## Class changes
+
+`EventHubClient` has been split into two clients - `EventHubProducerClient` (for sending
+messages) and `EventHubConsumerClient` (for subscribing to messages).
+
+| In v2                                       | Equivalent in v5                                                 | Sample |
+|---------------------------------------------|------------------------------------------------------------------|--------|
+| `EventHubClient.createFromConnectionString` | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [All](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/)
+| `EventHubClient.receive`                    | `EventHubConsumerClient.subscribe`                               | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEvents.ts) |
+| `EventHubClient.receiveBatch`               | Removed in favor of `EventHubConsumerClient.subscribe`           | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEvents.ts) |
+| `EventHubClient.createSender`               | `new EventHubProducerClient`                          | [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts) |
+| `EventHubClient.send`                       | `EventHubConsumerClient.sendBatch`                               | [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts) |
```

Comment:
```suggestion
| `EventHubClient.send`                       | `EventHubConsumerClient.sendBatch`                               | [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts) which requires you to create a batch of events first.|
```

Reply:
You might be thinking of receiveEvents (not sendEvents.ts which does the sending).

There's a comment block in receiveEvents that walks you through running sendEvents to prime your EventHub.

Reply:
My suggestion was more for the method `sendBatch` and not regarding the samples.

Current table contents tell the user that if they have been using `send`, they should now use `sendBatch`. But they also need to create a batch first, which they didnt need to do before. So, I believe its worthwhile calling that out.